index.html

```diff
@@ -152,8 +152,49 @@ <h3>Skills</h3>
         </ul>
       </li>
     </ul>
-
   </section>
+  <form action="https://formspree.io/f/mayaljan" method="post" id="form">
+    <fieldset class="contact">
+      <h2 class="contactheader">Get Started</h2>
+      <p class="intro">Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit,
+        sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      <label for="name"></label>
+      <input type="text" class="name" id="name" name="name" placeholder="Your Name" required />
+      <br>
+      <label for="name"></label>
+      <input type="text" class="name" id="email" name="email" value="Your Email" required />
+      <br>
+      <label for="message"></label>
+      <textarea name="message" class="message" id="message" placeholder="Enter your message here..."></textarea>
+      <input type="submit" class="btn1  " value="Start collaboration" />
+    </fieldset>
+  </form>
+  <footer>
+    <div  class="resume">
+      <p>Get my resume</p>
+    </div>
+    <ul class="social">
+      <li class="socialicon">
```

Comment:
It would be better if the icons were functional. This means that once the user has clicked they get the social media page on each one of the icons.